content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -36,11 +36,86 @@ are supported. If you are on an older version, please upgrade to 1.1 first.
 [Download the new Istio release](/docs/setup/kubernetes/#downloading-the-release)
 and change directory to the new release directory.
 
+### Istio CNI upgrade
+
+Choose one of the following **mutually exclusive** options to check whether
+[Istio CNI](/docs/setup/kubernetes/additional-setup/cni/) is installed and to update it:
+
+{{< tabset cookie-name="controlplaneupdate" >}}
+{{< tab name="Kubernetes rolling update" cookie-value="k8supdate" >}}
+
+You can use Kubernetes’ rolling update mechanism to upgrade the Istio CNI components.
+This is suitable for cases where `kubectl apply` was used to deploy Istio CNI.
+
+1. To check whether `istio-cni` is installed, search for `istio-cni-node` pods
+   and in which namespace they are running (typically, `kube-system` or `istio-system`):
+
+    {{< text bash >}}
+    $ kubectl get pods -l k8s-app=istio-cni-node --all-namespaces
```

Comment:
can you confirm in 1.1, the pod is labelled as `k8s-app=istio-cni-node`?  my env doesn't have CNI, and I know this is not typically how we label istio control plane components.

```
$ kubectl get pods -l app=pilot --all-namespaces       
NAMESPACE      NAME                           READY   STATUS    RESTARTS   AGE
istio-system   istio-pilot-5dd7b5f674-zrlvm   2/2     Running   0          25h
```

Reply:
The Helm chart has always used `k8s-app: istio-cni-node`, cf. its creation in https://github.com/istio/cni/commit/0f8609fa8c8647aa3234eb801e498f60129f7986 on 2019-10-04.

The only question mark I have is that the Helm chart was copied and modified from an older [.../install/manifests/istio-cni.yaml](https://github.com/istio/cni/blob/master/deployments/kubernetes/install/manifests/istio-cni.yaml), which uses `istio-node` instead of `istio-cni-node` as the label value and the name of the daemonset.

This one is referenced in istio/cni's `README.md`, but not in the Istio docs.

Reply:
I'll just remove [.../install/manifests/istio-cni.yaml](https://github.com/istio/cni/blob/master/deployments/kubernetes/install/manifests/istio-cni.yaml) and update README.md to avoid any confusion.

Reply:
FYI https://github.com/istio/cni/pull/130